client/maps/map-preview.tsx

```diff
@@ -77,16 +78,16 @@ const ZoomableMapImageContainer = styled.div`
   border-radius: 4px;
 `
 
-const StyledMapInfoImage = styled(MapInfoImage)<{ $zoom: number }>`
+const StyledMapInfoImage = styled(MapInfoImage)`
   ${fastOutSlowInShort};
 
-  transform: scale(${props => props.$zoom});
+  transform: scale(var(--sb-map-image-zoom));
```

Comment:
For future reference, the `--sb-map-image-` type naming is generally for things that are meant to be customizable by outside components, if you're just passing values internally you can do `--_foo` and stuff (but I don't really care that much, it just makes the code a bit less annoying to write)